Tools/Scripts/webkitpy/port/win_unittest.py

```diff
@@ -35,33 +35,35 @@
 from webkitpy.common.system.systemhost_mock import MockSystemHost
 from webkitpy.common.version_name_map import PUBLIC_TABLE, VersionNameMap
 from webkitpy.port import port_testcase
-from webkitpy.port.win import WinPort
+from webkitpy.port.win import WinCairoPort
 from webkitpy.tool.mocktool import MockOptions
 
 
 class WinPortTest(port_testcase.PortTestCase):
     os_name = 'win'
     os_version = Version.from_name('XP')
     port_name = 'win-xp'
-    port_maker = WinPort
+    port_maker = WinCairoPort
 
+    maxDiff = None
     def _assert_search_path(self, expected_search_paths, version, use_webkit2=False):
         port = self.make_port(port_name='win', os_version=version, options=MockOptions(webkit_test_runner=use_webkit2))
-        absolute_search_paths = list(map(port._webkit_baseline_path, expected_search_paths))
-        self.assertEqual(port.baseline_search_path(), absolute_search_paths)
+        platform_dir = port.host.filesystem.dirname(port._webkit_baseline_path("xxx"))
+        relative_search_paths = [port.host.filesystem.relpath(p, platform_dir) for p in port.baseline_search_path()]
+        self.assertEqual(relative_search_paths, expected_search_paths)
 
     def test_baseline_search_path(self):
-        self._assert_search_path(['win-xp', 'win-vista', 'win-7sp0', 'win-8', 'win-8.1', 'win-win10', 'win', 'mac'], Version.from_name('XP'))
-        self._assert_search_path(['win-vista', 'win-7sp0', 'win-8', 'win-8.1', 'win-win10', 'win', 'mac'], Version.from_name('Vista'))
-        self._assert_search_path(['win-7sp0', 'win-8', 'win-8.1', 'win-win10', 'win', 'mac'], Version.from_name('7sp0'))
+        self._assert_search_path(['wincairo-xp-wk1', 'wincairo-xp', 'wincairo-vista-wk1', 'wincairo-vista', 'wincairo-7sp0-wk1', 'wincairo-7sp0', 'wincairo-8-wk1', 'wincairo-8', 'wincairo-8.1-wk1', 'wincairo-8.1', 'wincairo-win10-wk1', 'wincairo-win10', 'wincairo-wk1', 'wincairo'], Version.from_name('XP'))
+        self._assert_search_path(['wincairo-vista-wk1', 'wincairo-vista', 'wincairo-7sp0-wk1', 'wincairo-7sp0', 'wincairo-8-wk1', 'wincairo-8', 'wincairo-8.1-wk1', 'wincairo-8.1', 'wincairo-win10-wk1', 'wincairo-win10', 'wincairo-wk1', 'wincairo'], Version.from_name('Vista'))
+        self._assert_search_path(['wincairo-7sp0-wk1', 'wincairo-7sp0', 'wincairo-8-wk1', 'wincairo-8', 'wincairo-8.1-wk1', 'wincairo-8.1', 'wincairo-win10-wk1', 'wincairo-win10', 'wincairo-wk1', 'wincairo'], Version.from_name('7sp0'))
```

Comment:
WinCairo dropped WK1 support. I think we no longer need to test wk1.

Reply:
I think explicitly dropping WK1 is reasonable, but I'd rather we just kept this PR as small as possible (we can presumably _also_ drop Windows XP), but changing that default here is a larger change!